lib/tracker_api/endpoints/search.rb

```diff
@@ -0,0 +1,53 @@
+module TrackerApi
+  module Endpoints
+    class Search
+      attr_accessor :client
+
+      def initialize(client)
+        @client = client
+      end
+
+      def get(project_id, query, params={})
```

Comment:
**TODO:** Add method documentation. This shouldn't be too verbose. Pivotal Tracker allows [just about anything](https://www.pivotaltracker.com/help/articles/advanced_search/).